src/sentry/static/sentry/app/components/organizations/multipleEnvironmentSelector.jsx

```diff
@@ -207,15 +211,23 @@ class MultipleEnvironmentSelector extends React.PureComponent {
   }
 
   render() {
-    const {value} = this.props;
+    const {value, loadingProjects} = this.props;
     const environments = this.getEnvironments();
 
     const validatedValue = value.filter(env => environments.includes(env));
     const summary = validatedValue.length
       ? `${validatedValue.join(', ')}`
       : t('All Environments');
 
-    return (
+    return loadingProjects ? (
+      <StyledHeaderItem
+        data-test-id="global-header-environment-selector"
+        icon={<StyledInlineSvg src="icon-window" />}
+        loadingProjects
+      >
+        {t('Loading...')}
```

Comment:
Should this use a unicode ellipsis instead of three periods?

Reply:
yup sounds good ill make that change real quick